File: MPI/mpicheck/tests/functions/2.0/MPI_Open_port.f90

        
        program main
        use mpi
        
        INTEGER var_0
       CHARACTER*(10) var_1
       INTEGER var_2
        call mpi_open_port(var_0, var_1, var_2)
       call pmpi_open_port(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_Open_port.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Info var_0;
    char *var_1;
    int ret;
    /* calls */
    ret = MPI_Open_port(var_0, var_1);
    ret = PMPI_Open_port(var_0, var_1);
    return 0;
}
